packages/ui/alert/index.css

```diff
@@ -29,6 +29,10 @@
   background-color: var(--color-layout-error-background);
 }
 
+.info {
+  background-color: var(--color-layout-secondary-background);
```

Comment:
@ailincaibianca there is a border-radius associated with the `info`
 variant that is missing here:
 `border-radius: var(--gridSize)`

![Screenshot 2023-05-16 at 10 49 01](https://github.com/figshare/fcl/assets/6098942/b9895b0c-6bea-4636-a89a-09933d6e2cd1)